docs/integration-guides/websockets.md

```diff
@@ -0,0 +1,504 @@
+title: WebSockets | Nano Documentation
+description: Details for integration into WebSockets for notifications from the Nano node.
+
+!!! note ""
+    Available in version 19.0+ only. When upgrading from version 18 or earlier, the node performs a confirmation height upgrade. During this process, the WebSocket notifications may include confirmations for old blocks. Services must handle duplicate notifications, as well as missed blocks as WebSockets do not provide guaranteed delivery. Reasons for missed blocks include intermittent network issues and internal containers (in the node or clients) reaching capacity.
+
+--8<-- "multiple-confirmation-notifications.md"
+
+The Nano node offers notification of confirmed blocks over WebSockets. This offers higher throughput over the HTTP callback, and uses a single ingoing connection instead of an outgoing connection for every block.
+
+The HTTP callback is still available and both mechanisms can be used at the same time.
+
+**Example**
```

Comment:
By moving this out of the general Advanced guide we have more header levels to play with. I think configuration and example clients should have their own (sub?)sections, and the types of messages below could be placed under a general "Specification" section.

Reply:
Good call, made these adjustments